pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/SimpleConsumerWrapper.java

```diff
@@ -48,12 +44,17 @@
 import kafka.javaapi.consumer.SimpleConsumer;
 import kafka.javaapi.message.ByteBufferMessageSet;
 import kafka.message.MessageAndOffset;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.kafka.common.errors.TimeoutException;
+import org.apache.kafka.common.protocol.Errors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 /**
  * Wrapper for Kafka's SimpleConsumer which ensures that we're connected to the appropriate broker for consumption.
  */
-public class SimpleConsumerWrapper implements Closeable {
+public class SimpleConsumerWrapper implements Closeable, IPinotKafkaConsumer {
```

Comment:
ah now i see why you chose to use SimpleConsumerWrapper as the class name (instead of SimpleConsumer) -- so you can minimize changes. 

OK, go for it,